dotnet/src/SemanticKernel.Core/Functions/KernelFunctionFromMethod.cs

```diff
@@ -206,7 +206,7 @@ private record struct MethodDetails(string Name, string Description, Implementat
         IReadOnlyList<KernelParameterMetadata> parameters,
         KernelReturnParameterMetadata returnParameter,
         ReadOnlyDictionary<string, object?>? additionalMetadata = null) :
-        base(functionName, pluginName, description, parameters, returnParameter, properties: additionalMetadata)
+        base(functionName, pluginName, description, parameters, returnParameter, additionalMetadata: additionalMetadata)
```

Comment:
I'd keep the naming consistent either `additionalMetadata` or `additionalProperties` everywhere

Reply:
I think context matters.  Outside of the metadata class AdditionalMetadata makes sense, since it's clarifying where the data will end up. However inside the metadata class the word metadata is unnecessary and properties make more sense.